src/cm-plugins/download/cm3-download-bundle.js

```diff
@@ -88,46 +83,55 @@ const initMap=function(store){
 
   map.addInteraction(selectSingleClick);
 
-  selectSingleClick.on('select', function(evt) {
-    
-    console.log(Object.getOwnPropertyNames(evt));
-
-    const state = evt.selected[0].values_.NAME
-    const statefips = evt.selected[0].values_.STATEFP
-
-    // extent_ returns 2 pairs of coordinates
-    // averaging the 2 pairs returns the center of selected polygon
-    const extentCoords = evt.selected[0].values_.geometry.extent_
-    const extentCoords0 = [0,1].map(x=>extentCoords[x])
-    const extentCoords1 = [2,3].map(x=>extentCoords[x])
-    const overlayCoord = [(extentCoords0[0]+extentCoords1[0])/2,(extentCoords0[1]+extentCoords1[1])/2]
-
-    // pop-up on click
-    content.innerHTML = '<p>Download structure data for ' + state + '?</p>';
-    overlay.setPosition(overlayCoord);
+  selectSingleClick.on('select', function (evt) {
 
     // Reset popup
-    function closeDownPopUp () {
+    function closeDownPopUp() {
       overlay.setPosition(undefined);
       closer.blur();
       selectSingleClick.getFeatures().clear(); // clear selected state
       return false;
     }
 
-    // Map closeDownPopUp to cancel button
-    closer.onclick = closeDownPopUp;
+    // if clicked on the state mask, evt is a SelectEvent obj
+    // if clicked on vtl, evt is a RenderFeature obj
+    // RenderFeature is not compatible with Select(), solution is to switch to 
+    // https://github.com/openlayers/openlayers/issues/9840
```

Comment:
Good reference. This seems interesting. I will download source and test it out later - thanks for researching this.